### PR TITLE
Remove TCC related code

### PR DESCRIPTION
**Pull request by ttytm**
Makes the `Makefile` a lot cleaner as different compilers only need to be handled for Linux now.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,9 +102,6 @@ Think of WebUI like a WebView controller, but instead of embedding the WebView c
   # GCC
   mingw32-make
   
-  # TCC
-  mingw32-make COMPILER=tcc
-
```
Comment (ttytm) [inline README.md:107]: What is left in the readme is:
> - Lightweight (~160 Kb using TCC) & Small memory footprint

How should this be updated? Didn't knew at first glance what things are calculated.

Comment (hassandraga) [reply README.md:107]: We can keep it as the user can still add the headers files and compile the lib using TCC to get the 160Kb lib. But let's change it to `~200Kb` as GCC/Clang produces 200Kb files.

The main goal here is to explain to the user that WebUI is a KBS lib and not MBs lib compared to other GUI libraries.

**Review by hassandraga — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ttytm) [reply README.md:107]: So you say?
Lightweight (~200 Kb using GCC/Clang) & Small memory footprint

Or
Lightweight (~200 Kb using TCC) & Small memory footprint

Comment (hassandraga) [reply README.md:107]: Lightweight _~200 Kb_ & Small memory footprint

**Pull request merged**